.evergreen/serverless/create-instance.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
Are these files copied from somewhere?

Reply:
Yes, from here - https://github.com/mongodb-labs/drivers-evergreen-tools/tree/master/.evergreen/serverless

Reply:
We have d-e-t as a submodule in https://github.com/mongodb/mongo-ruby-driver/tree/master/.mod. You can then symlink into it like https://github.com/mongodb/mongo-ruby-driver/blob/master/spec/support/ocsp does.

Reply:
Oh, missed that, thanks. Fixed.